### ISSUE TITLE
secureTextEntry changes font size in input with stackedLabel

### ISSUE THREAD

**Issue by Nikos410** [GeekyAnts/NativeBase#2922]
## I have gone through these following points

- [x] Check latest documentation: https://docs.nativebase.io/
- [x] Check for existing open/closed issues for a possible duplicate before creating a new issue: https://github.com/GeekyAnts/NativeBase/issues
- [x] Use the latest NativeBase release: https://github.com/GeekyAnts/NativeBase/releases
- [x] Check examples from NativeBase KitchenSink https://github.com/GeekyAnts/NativeBase-KitchenSink
- [x] For discussion purpose make use of NativeBase Slack: http://slack.nativebase.io/
- [x] For queries related to theme, check [Theme Variables](https://docs.nativebase.io/docs/ThemeVariables.html) from Docs and live NativeBase Theme Editor http://nativebase.io/customizer/

#### node, npm, react-native, react and native-base version, expo version if used, xcode version

node v10.16.0
react-native v0.60.5
react v16.8.6
native-base v2.13.7

android 7.1.2

#### Expected behaviour

The text in a normal input should have the same font size as the text in an input with `secureText={true}`

#### Actual behaviour

The text in a normal input has a larger font size as the text in an input with `secureText={true}`

#### Steps to reproduce

EDIT: I just noticed this only happens with `stackedLabel`, with all other labels the font sizes are the same.

Add a normal input and one input with `secureTextEntry={true}`, both with `stackedLabel`

```typescript
import { Container, Content, Footer, Form, Input, Item, Label } from 'native-base';
import React from 'react';

export interface Props {}

export default class App extends React.PureComponent<Props> {
    public constructor(props: Props) {
        super(props);
    }

    public render() {
        return (
            <Container>
                <Content>
                    <Form>
                        <Item stackedLabel={true}>
                            <Label>Username</Label>
                            <Input />
                        </Item>

                        <Item stackedLabel={true}>
                            <Label>Password</Label>
                            <Input secureTextEntry={true} />
                        </Item>
                    </Form>
                </Content>
                <Footer />
            </Container>
        );
    }
}
```
![Screenshot_20190909-100657](https://user-images.githubusercontent.com/20876882/64514047-2cf5d780-d2ea-11e9-954e-f3783c70eb56.png)

#### Is the bug present in both iOS and Android or in any one of them?

I'm only developing for android right now, so I could not try to reproduce the issue for iOS, sorry.

**Comment by sankhadeeproy007** [comment]
@Nikos410 Can you try using TextInput from react-native and see if the problem persists?

**Comment by Nikos410** [comment]
@sankhadeeproy007 If I use TextInput, the font size in both fields is the same, but it is smaller than the text in native-base Inputs and the text is centered in the middle of the input. 

It looks like as if I use secureTextEntry the input gets rendered as a react-native TextInput instead of a native-base Input.

**Comment by sankhadeeproy007** [comment]
@Nikos410 Thanks for the detailed description. Will take a look at it.

**Comment by Nikos410** [comment]
@sankhadeeproy007 The TextInputs also seem to be affected by which label I'm using. 

* inlineLabel / placeholderLabel: Both inputs have small font size, text is aligned left, secureTextEntry works (dots shown)
* floatingLabel: Both inputs have large font size, text is aligned left, secureTextEntry does not work (Text is visible)
* stackedLabel: Both inputs have small font size, text is centered, secureTextEntry works

**Comment by sankhadeeproy007** [comment]
Fixed in 2.13.8
